examples/macroquad-ui/src/main.rs

```diff
@@ -1,6 +1,7 @@
+#[cfg(target_os = "android")]
+use crossbow_permissions::request_permission::ask_for_permission;
```

Comment:
It's better to make a single cross-platform function. Don't add `cfg target_os` here

Reply:
Ok. It was added because of rust-analyzer issues.